docs/content/functioneel/zaaktypen_api.md

```diff
@@ -0,0 +1,182 @@
+| Domein          | Zaakgericht werken                                                                                                                                                                                |
+| Provider        | [Zaaktypecataloguscomponent](https://www.gemmaonline.nl/index.php/GEMMA2/0.9/id-3ef9cdd9-631c-4d3e-88c3-f756423d6314) (GEMMA2)                                                              |
+| Consumer        | Componenten waarmee zaaktypen beheerd worden en componenten die zaaktypegegevens gebruiken voor het behandelen van zaken, zoals Zaakafhandelcomponenten (ZAC’s).                                                              |
+| Informatiemodel | [IMZTC, versie 2.2](https://www.gemmaonline.nl/images/gemmaonline/1/1d/EgIM_20180621_-_ag_4_-_GEMMA_ImZTC_2.2_-_CONCEPT_20180613.zip) (in-ontwikkeling)                                         |
+| Specificaties   | <https://ref.tst.vng.cloud/zrc/api/v1/schema>                                                                                                                                                     |
+| Bijzonderheden  | -                                                    |
+
+Deze API omvat (vooralsnog) de resources Catalogussen, Zaaktypen, Informatieobjecttypen, Roltypen, Statustypen en Zaakobjecten.
+
+Op dit moment bieden de resources alleen de mogelijkheid tot raadplegen van de gegevens (GET).
+
+### Resource: catalogus
+
+| **Aspect**     | **Beschrijving**                                                                                                                                                                 |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil een lijst van alle beschikbare CATALOGUSsen of een specifieke catalogus opvragen.         |
+| *Toevoegen*    |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Wijzigen*     |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Verwijderen*  |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Raadplegen*   |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | Een lijst met beschikbare catalogussen wordt teruggegeven aan de consumer bij het opvragen van de lijst.<br>Een specificieke catalogus wordt teruggegeven aan de consumer. |
+| Gegevens       | objecttype CATALOGUS inclusief een lijst met relaties naar daarin gedefinieerde ZAAKTYPEn.<br>Zie de volgende tabel voor de gegevens van deze resource                                 |
+| Specificaties  | https://ref.tst.vng.cloud/ztc/api/v1/schema/#tag/catalogus                                                                                                   |
```

Comment:
https://ref.tst.vng.cloud/ztc/api/v1/schema/#tag/catalogussen lijkt de correcte URL te zijn

Reply:
Maar ik zie geen schema van het objecttype catalogus, alleen van de response bij een 200. Is dat dan hetzelfde schema als gebruikt wordt wanneer ik (later, is nu nog niet ingebouwd) een catalogus toe zou voegen via een POST?

Reply:
Jep. REST - state van een resource, je gooit dus altijd dezelfde resource over de lijn in beide richtingen (kort door de bocht, maar bijna correct).

Reply:
Dat snap ik wel maar het gaat er om dat bij het beschrijven van een resource naar een operatie verwezen wordt. Terwijl die resource straks bij verschillende operaties van toepassing is. 
Om de verwijzing (in de documentatie) zuiver te houden moet in de documentatie verwezen worden naar een schema van de betrokken resource, niet naar een operatie waar toevallig ook een vergelijkbaar schema in gebruikt wordt.

Een andere mogelijkheid is per operatie een verwijzing te maken, alleen durf ik niet te garanderen dat het er dan duidelijker op wordt

Reply:
In de Zaken API documentatie heb ik bij wijze van experiment een verwijzing per operatie toegevoegd. Persoonlijk vind ik dat duidelijker.